samples/bookinfo/platform/kube/bookinfo.yaml

```diff
@@ -78,6 +78,13 @@ spec:
         - containerPort: 9080
         securityContext:
           runAsUser: 1000
+          allowPrivilegeEscalation: false
```

Comment:
At a minimum, I think we would wome some comments here, to explain that the following is optional, only needed if you are deploying in a PSA enabled cluster?

Reply:
> Do we really want this added complexity in the bookinfo deployment, which is the simple getting started with Istio example? We don't want new users to think their deployments can't run in Istio without these things. Would it be better to have a separate PSA version of the deployment, that is only used in the PSA doc?
> 
> Thoughts?

Do we know the impact of these additional settings when deployed on a non-PSA-enabled cluster? I.e., will anything break, or will things continue to work? If stuff breaks, the decision to use a separate PSA deployment version is clear. 

But even if there's no impact, I'd say keeping things separate makes it cleaner (and, I might be wrong, but I don't think there should be a high maintenance cost of having another .yaml file)

Reply:
I think comments here can be useful. I'm not sure if the average Istio users look at these demos to find a minimum set of requirements for their applications, so I will defer to you on this. I'll update this PR accordingly once a consensus has been reached.